app/Models/Group.php

```diff
@@ -180,19 +106,14 @@ public function chooseYear($year)
 
     public function chosenStateKey()
     {
-        // Root group's session
-        return sprintf('group/%s/state', $this->root()->id);
+        return sprintf('group/%s/state', $this->id);
     }
     public function chosenState()
     {
-        $state_abbr = session($this->chosenStateKey(), $this->root()->state_abbr);
-
-        if ($state_abbr)
-        {
-            return State::where('state_abbr', $state_abbr)->first();
-        }
-
-        return null;
+        $state_abbr = session($this->chosenStateKey(), $this->state_abbr);
```

Comment:
```
$variableName
ClassTraitOrInterfaceName
$className->property_name
$className->methodName()
global_function_call()
```

Should be

```
$stateAbbr
```

I believe if you run `composer cs && composer phpmd` it will tell you about these things and enforce the standards for you.

Reply:
Got it... fixed!